mappings/net/minecraft/world/chunk/light/ChunkLightProvider.mapping

```diff
@@ -42,11 +42,13 @@ CLASS net/minecraft/class_3558 net/minecraft/world/chunk/light/ChunkLightProvide
 		ARG 5 direction
 		ARG 6 opacity2
 	METHOD method_20479 getStateForLighting (JLorg/apache/commons/lang3/mutable/MutableInt;)Lnet/minecraft/class_2680;
-		ARG 1 pos
-	METHOD method_20599 setRetainData (Lnet/minecraft/class_1923;Z)V
+		ARG 1 blockPos
```

Comment:
I think it's better to leave these as `pos` rather than `blockPos`.

Reply:
Right- I guess it might be clear enough from the method name. Though just here where there's so many different kinds of `pos`, where they're all `longs`, I think it could still be useful to clarify everything? Not sure

Reply:
```suggestion
		ARG 1 pos
```

Reply:
I still think here it's best to clarify everywhere since there's so many different pos parameters that are all longs

Reply:
It returns a `BlockState`, and you can infer it's a block pos